WearTilesKotlin/app/src/debug/java/com/example/wear/tiles/golden/Meditation.kt

```diff
@@ -81,46 +87,25 @@ object Meditation {
                         setWidth(expand())
                         setHeight(expand())
                         addContent(
-                            button(
-                                onClick = clickable(),
-                                width = expand(),
-                                height = expand(),
-                                colors = filledTonalButtonColors(),
-                                style =
-                                if (isLargeScreen()) {
-                                    defaultButtonStyle()
-                                } else {
-                                    smallButtonStyle()
-                                },
-                                iconContent = {
-                                    icon(
-                                        context.resources.getResourceName(
-                                            R.drawable.outline_air_24
-                                        )
+                            meditationButton(
+                                MeditationTask(
+                                    label = "Breath",
+                                    iconResourceIdName =
+                                    context.resources.getResourceName(
+                                        R.drawable.outline_air_24
                                     )
-                                },
-                                labelContent = { text("Breath".layoutString) }
+                                )
                             )
                         )
                         addContent(ButtonGroupDefaults.DEFAULT_SPACER_BETWEEN_BUTTON_GROUPS)
                         addContent(
-                            button(
-                                onClick = clickable(),
-                                width = expand(),
-                                height = expand(),
-                                colors = filledTonalButtonColors(),
-                                style =
-                                if (isLargeScreen()) {
-                                    defaultButtonStyle()
-                                } else {
-                                    smallButtonStyle()
-                                },
-                                iconContent = {
-                                    icon(context.resources.getResourceName(R.drawable.ic_yoga_24))
-                                },
-                                labelContent = {
-                                    text("Daily mindfulness".layoutString, maxLines = 2)
-                                }
+                            meditationButton(
+                                MeditationTask(
+                                    label = "Daily mindfulness",
+                                    iconResourceIdName =
+                                    context.resources.getResourceName(R.drawable.ic_yoga_24),
```

Comment:
Is this resource naming approach something in samples? or should we call it out as good practice?

Reply:
See separate comment.